src/frameworks/index.ts

```diff
@@ -503,6 +503,27 @@ export async function prepareFrameworks(
       packageJson.engines ||= {};
       packageJson.engines.node ||= NODE_VERSION;
 
+      for (const [name, version] of Object.entries(
+        packageJson.dependencies as Record<string, string>
+      )) {
+        if (version.startsWith("file:")) {
+          const path = version.split(":")[1];
```

Comment:
let's get more strict here, `version.replace(/^file:/, '')`, we probably need to be smarter about relative paths too

Reply:
@jamesdaniels what specific concerns do you have about relative paths?

Reply:
if it's relative, e.g, `file:../foo` we probably want to resolve it relative to the source directory rather than the cwd, which i believe is what the default is

Reply:
in my initial testing, relative paths seem to be working as intended without any modifications. there might be some scenarios I missed but things like `file:../foo.js` and `file:../test_dir` were resolved and copied/packed correctly.